data/styles/clear/style-clear/symbols/wind_turbine-m.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg version="1.1" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
```

Comment:
let's have `width` and `height` specified so that it shows up in a true size in GH review and for consistency also

Reply:
@RedAuburn